docs/source/tutorials/base/how_to_train/detection.rst

```diff
@@ -0,0 +1,382 @@
+Object Detection model
+======================
+
+This tutorial reveals end-to-end solution from installation to model export and optimization for object detection task on a certain example.
+On this page we show how to train, validate, export and optimize ATSS model on WGISD public dataset.
+
+.. note::
```

Comment:
How about making a bigger and bold font for this sentence?
![image](https://user-images.githubusercontent.com/80735344/213086206-75a9aa7e-a2e9-4f81-9925-b4feb4e396fd.png)

Reply:
It already is. In the next PR I'll leave an instruction, how to build this docs, my bad
![image](https://user-images.githubusercontent.com/33455576/213126691-719d7e2f-0c9d-4ebd-8705-2b0831a57c4a.png)